docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -807,6 +808,89 @@ END;
+      <tgroup cols="2">
+        <colspec align="left" colwidth="122.40pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis role="bold">LZW</emphasis></entry>
+
+            <entry>The default compression. It is a variant of the
+            Lempel-Ziv-Welch algorithm. This is the original index compression
+            format (which is less efficient than the inplace compression
+            format). It remains the default for backward
+            compatibility.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">ROW</emphasis></entry>
+
+            <entry>Compresses index entries based on differences between
+            contiguous rows (<emphasis role="bold">for use with fixed-length
+            records only</emphasis>). It is recommended for use in
+            circumstances where speedier decompression time is more important
+            than the amount of compression achieved. This is available as an
+            alternative to the original LZW compression, but does not use the
+            'inplace' compression format.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">'inplace'</emphasis></entry>
+
+            <entry>Causes the index to be built using the inplace lz4
+            compression format.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4hc'</emphasis> </emphasis></entry>
+
+            <entry>Causes inplace compression on the key fields and lz4hc
+            compression on the payload. The resulting index can be
+            significantly smaller, but can take much longer to create.</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>The inplace index compression format (introduced in version 9.2.0)
+    is more efficient and faster. Generally, the inplace index format improves
+    compression of branch nodes in the btree and allows them to be searched
+    without decompression. The original LZW index compression implementation
+    decompresses the rows when they are read from disk.</para>
+
+    <para>With the inplace format, the btree is created from compressed data
+    and therefore more search terms fit into memory which can greatly improve
+    btree search performance. In addition, inplace compression is better than
+    LZW, also increasing the number of leaf nodes that can fit into memory.
+    This can significantly reduce disk access by increasing the number of
+    branch pages held in memory. It is also significantly faster at
+    decompressing leaf pages.</para>
+
+    <para>Whether performance is better with lz4hc (a high-compression variant
+    of lz4) on the payload fields depends on the access characteristics of the
+    data and how much of the index is cached in memory.</para>
+
+    <para>You should review your existing indexes. If you have fields that are
```

Comment:
This shouldn't be related to this attribute the LRM.  If it was included it should be a comment on indexes in general.